scripts/aws_bastion_mfa_export.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+while getopts p: flag
+do
+    case "${flag}" in
+        p) profile=${OPTARG};;
+    esac
```

Comment:
Would a default covering like
*  ) echo "Unimplemented option: -$OPTARG" >&2; exit 1;;

Be something good here?

Reply:
I added some changes based on your feedback.

These are the covered scenarios so far:

```
$ bash aws_bastion_mfa_export.sh
no argument flags provided

$ bash aws_bastion_mfa_export.sh -a
Invalid option -a

$ bash aws_bastion_mfa_export.sh -p
Invalid option -p requires argument
```